src/tox_current_env/hooks4.py

```diff
@@ -146,7 +148,19 @@ def _get_python(self, base_python):
         )
 
     def create_python_env(self):
-        return None
+        # Fake Python environment just to make sure all possible
+        # commands like python or python3 works.
+        self.tempdir = tempfile.TemporaryDirectory()
+        for suffix in (
+            "",
+            f"{sys.version_info.major}",
+            f"{sys.version_info.major}.{sys.version_info.minor}",
+        ):
+            os.symlink(sys.executable, Path(self.tempdir.name) / f"python{suffix}")
+        os.environ["PATH"] = ":".join((os.environ["PATH"], str(self.tempdir.name)))
+
+    def _teardown(self):
+        del self.tempdir
```

Comment:
Similarly, if we actually need to patch `os.environ`, we should restore it here.

Reply:
Good point, will do.